linux_os/guide/system/bootloader-grub2/uefi/uefi_no_removeable_media/rule.yml

```diff
@@ -30,12 +30,14 @@ ocil_clause: 'it is not'
 
 ocil: |-
     To verify the system is not configured to use a boot loader on removable media,
-    run the following command:
+    check that the grub configuration file has the set root command in each menu
+    entry with the following commands:
+    <pre>$ sudo grep -cw menuentry {{{ grub2_uefi_boot_path }}}/grub.cfg</pre>
```

Comment:
There should be mentioned that `grep -c` only prints a number which is count of the `menuentry` word occurrences.

Reply:
Mentioned this on both rules after this line. Does it look good?